packages/react-router/src/runtime.ts

```diff
@@ -0,0 +1,35 @@
+import { AbstractRuntime, isNil } from "@squide/core";
+
+import { NavigationItemRegistry } from "./navigationItemRegistry.ts";
+import type { RootNavigationItem } from "./navigationItemRegistry.ts";
+import type { RootRoute } from "./routeRegistry.ts";
+import { RouteRegistry } from "./routeRegistry.ts";
+
+export class Runtime extends AbstractRuntime<RootRoute, RootNavigationItem> {
+    protected _routeRegistry = new RouteRegistry();
+    protected _navigationItemRegistry = new NavigationItemRegistry();
+
+    registerRoutes(routes: RootRoute[]) {
+        if (!isNil(routes)) {
```

Comment:
if you check for undefined routes, i would expect the typing to accept undefined.

Reply:
You're right, let's remove this check!